book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    return if @rentals.include?(rental)
+
+    @rentals << rental
+  end
```

Comment:
- [ ] (Optional) Good job adding the add_rental method. Your approach assumes that rentals are created elsewhere and focuses on managing a collection of existing Rental objects. Kindly consider using the approach I have proposed below as it is a little more direct.

e.g
```ruby
def add_rental(person, date) 
    Rental.new(date, self, person)
  end
```
 .